src/Globe/TileMesh.js

```diff
@@ -66,77 +67,50 @@ function TileMesh(params, builder, geometryCache) {
 
     // Layer
     this.setDisplayed(false);
-}
 
-TileMesh.prototype = Object.create(NodeMesh.prototype);
+    this.layerUpdateState = {};
+}
 
+TileMesh.prototype = Object.create(THREE.Mesh.prototype);
 TileMesh.prototype.constructor = TileMesh;
 
-TileMesh.prototype.buildHelper = function buildHelper() {
-    // TODO Dispose HELPER!!!
-    var text = (this.level + 1).toString();
-
-    var showHelperBox = true;
-
-    if (showHelperBox)
-        { this.helper = new OBBHelper(this.geometry.OBB, text); }
-    else
-        { this.helper = new SphereHelper(this.geometry.boundingSphere.radius); }
-
-    if (this.helper instanceof SphereHelper)
-
-        { this.helper.position.add(new THREE.Vector3().setFromMatrixPosition(this.matrixWorld)); }
-
-    else if (this.helper instanceof OBBHelper)
-
-        { this.helper.translateZ(this.distance); }
-
-    this.link.add(this.helper);
-};
-
 TileMesh.prototype.dispose = function dispose() {
-    // TODO à mettre dans node mesh
     this.material.dispose();
     this.geometry.dispose();
     this.geometry = null;
     this.material = null;
 };
 
-TileMesh.prototype.setUuid = function setUuid(uuid) {
-    this.id = uuid;
```

Comment:
yes, I will prepare PR from this [commit](https://github.com/iTowns/itowns2/pull/214/commits/86d76ec5ca017b6ce3935b67c23363c29aa2eb71) where I could fix select tile